src/common/header/containers/HeaderContainer.tsx

```diff
@@ -46,6 +47,9 @@ export function HeaderContainer({ onSearchChange }: HeaderProps): JSX.Element {
         <GridItem colStart={3}>
           <TermButtons />
         </GridItem>
+        <GridItem colStart={4}>
+          <GitHubButton />
+        </GridItem>
```

Comment:
Technically speaking, this grid only has 3 columns defined (line 23) so this behaviour isn't exactly clear despite doing what we probably want. I'd maybe suggest putting it into the block where `TermButtons` is defined. You'll likely need to add a flexbox and set the attributes so the buttons and GitHub link space evenly.

Reply:
Makes sense! I've [committed a change for that](https://github.com/VikeLabs/courseup/pull/257/commits/cce3aab0b69d5a6b251bc0ae1b7cd9c270cfd7d2).